### ISSUE TITLE
[ADE Support] Debt Resolution: One VA Debt Letter - PDF Accessibility (Tag Markup)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#104267]
> [!NOTE]
> **QUICK SUMMARY:** 
> The Debt Resolution Team is looking to get a head start on PDF accessibility as they approach the start of the Collab. Cycle. They are wanting to mark up the example PDF to be provided by Denise with the expected tags so it is accessible and 508 compliant. Team is pushing for a PDF and they want it fast as they have an SES Goal for their leadership that says it will be **live** **in** **April**.  

## Description of the Work

This is the child ticket of #104259 and was created to track the PDF audit and markup for the Debt Resolution Team, who is rolling this out live in April 2025. 

#### Initiative
_Stakeholders at the Debt Management Center (DMC) have asked that we establish what we are calling the "One VA Debt Letter" to comply with legislation._

#### Desired outcome
_Mark up the example PDF to be provided by Denise with the expected tags so it is accessible and 508 compliant_

**From Denise Coveyduc:** 

> Our stakeholders at the Debt Management Center (DMC) have asked that we establish what we are calling the "One VA Debt Letter" to comply with legislation.
> 
> They are pushing for a PDF and they want it fast as they have an SES Goal for their leadership that says it will be live in April. 
> 
> I am fully expecting there to be a few iterations of this but I am hopeful we can achieve this goal for them.
> The team currently has a PDF implementation that I don't think will be accessible; at least not out of the gates.  They are exploring other avenues that are 508 accessible but given the timeline they are much more time intensive and complicated. I guess I am hoping we can launch with this initial version and iterate to improve a11y.
> 

#### Why would I ask for this?

> 1. This is a HUGE initiative for our partners who report to Congress on this every 3 months and they are getting a lot of flack that it hasn't been done
> 2. Veterans already have ALL of this information available on the debt portal today this is simply adding a button that will allow them to generate a PDF to download/save this consolidated view of the information.
> 3. Veterans will continue to receive the debt letters that notify them of overpayment debts and statements that notify them about medical copays. This online version is a combined view of that information and in addition.
> 4. It isn't something Veterans are asking for and I am not even sure how useful it will be - we may end up taking it down entirely depending on the response and congressional feedback.

## About the Team 

- **Product name**: One VA Debt Letter
- **Team name**: Debt Resolution - Financial Management
- **OCTO product owner**: @denisecoveyduc 
- **Product manager**: @Tom-Davis 
- **Designer(s)**: @mgleo @josephrlee @charlottecesana 
- **Slack channel**: #financial-management
- **Dedicated accessibility specialist on your team** (if you have one):
- **Accessibility Champ** (if you have one): 

## Supporting Artifacts 
-  [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/one_va_debt_letter/product%20outline.md)
- **Denise will add the example PDF that the ADE team member can work with**
    -  **Current/ Draft PDF version** (that the Financial Management team is working on): [debt_letter_test_123.pdf](https://github.com/user-attachments/files/19060737/debt_letter_test_123.pdf) 
    -   **Reference**: This is the example they were given to replicate [example.pdf](https://github.com/user-attachments/files/19060755/example.pdf)
    - Please connect with [@digitaldrk](https://github.com/digitaldrk) and [@Dr-Pongo](https://github.com/Dr-Pongo) to discuss in more detail.
- [CC Kickoff Ticket #104021](https://github.com/department-of-veterans-affairs/va.gov-team/issues/104021)
- [ADE PDF GitHub Documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/11f3c67169a86b5c8602f109c2a1efdd3778737d/teams/ADE/create-accessible-PDF.md)

## ADE Tasks
> [!TIP]
> Customize ADE Tasks as needed. 
- [ ] Collaborate with Denise
- [x] Receive PDF current draft from team 3/3/2025 
- [x] Add draft PDF to supporting artifacts
- [x] Receive example to work from c/o team 3/3/2025
- [x] Add example to work from to supporting artifacts
- [ ] ADE Support by reviewing the PDF provided 
- [ ] Document compliance for accessibility and provide annotations/recommendations.
- [ ] Review and provide to Debt. Resolution Team

## Acceptance Criteria
> [!TIP]
> Customize ADE Acceptance Criteria as needed. 
- [ ] All PDFs are reviewed by ADE
- [ ] Compliance with accessibility feedback is received and relayed

### Ticket Updates
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi) added @SarahKay8 and @EvanAtCoforma due to PDF accessibility needs, and Evan is available after Wednesday 3/5/2025. 
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [x] **Read** Important Notifications (ADE Internal) at the bottom of this ticket.
- [x] **Delete** Important Notifications (ADE Internal) at the bottom of this ticket, after reading. 

<HR> 

### Related Tickets

<details><summary>Review important information related to connecting tickets</summary>

#### Tasks are Now Sub-Issues
- Related tickets are no longer being tracked in task lists, due to the planned depreceiation of tasks by GitHub in April 2025. 
- Instead, related tickets will be connected to tickets via sub-issues.
##### Connecting Tickets
- **To connect an existing ticket**, choose the drop-down from "create sub-issue" and select "add existing issue"
- **To connect a new issue,** choose the drop-down from "create sub-issue" and select "create sub-issue," and select the respective template. 

_Please delete this alert after reading and/or prior to saving this ticket template._

</details>

**Comment by sara-amanda** [comment]
## Additional Supporting Artifacts

I am going to add these to the supporting artifacts in this work ticket. They came over today c/o of the intake. 

- [x] **From**: [denisecoveyduc](https://github.com/denisecoveyduc) 
    -  https://github.com/department-of-veterans-affairs/va.gov-team/issues/104259#issuecomment-2695592057
        - > Here is the current/draft PDF version that the Financial Management team is working on. Please connect with             [@digitaldrk](https://github.com/digitaldrk) and [@Dr-Pongo](https://github.com/Dr-Pongo) to discuss in more detail. [debt_letter_test_123.pdf](https://github.com/user-attachments/files/19060737/debt_letter_test_123.pdf)
- [x] **From**: [digitaldrk](https://github.com/digitaldrk)
    - https://github.com/department-of-veterans-affairs/va.gov-team/issues/104259#issuecomment-2695598000
        - > FYI, this is the example we were given to replicate [example.pdf](https://github.com/user-attachments/files/19060755/example.pdf)

cc: @SarahKay8 @EvanAtCoforma @lakper

**Comment by SarahKay8** [comment]
@digitaldrk @Dr-Pongo 

Hi! There aren't any tags in the tags panel on either pdf. Am I submitting feedback on how I would tag/code these up?

**Comment by artsymartha68** [comment]
Yes please @SarahKay8 !

**Comment by SarahKay8** [comment]
Ok!

**Comment by digitaldrk** [comment]
> [@digitaldrk](https://github.com/digitaldrk) [@Dr-Pongo](https://github.com/Dr-Pongo)
> 
> Hi! There aren't any tags in the tags panel on either pdf. Am I submitting feedback on how I would tag/code these up?

Sorry I didn't respond. Yep, sounds like we're on the same page :)

**Comment by digitaldrk** [comment]
> Here's some markup notes I wrote down. Expected tools (Commonlook, Adobe, or PDFkit) [Debt Letter Markup Notes](https://docs.google.com/document/d/11UV3pr7WzSGfJzRHpgjoZ5hbQ1tsoZ2GDHkykJc9_Po/edit?tab=t.0) [@digitaldrk](https://github.com/digitaldrk)
> 
> Let me know if there's trouble accessing this (unfortunately, I don't have a Commonlook license otherwise I would've tagged this up for you all)

This gives us a great idea of what we need to do, thank you!

**Comment by mgleo** [comment]
Hi @SarahKay8 

@digitaldrk and @Dr-Pongo worked on a new version of this PDF using PDFKit. 
Can you take a look at this version? 
Thank you

[PDFKit_one_debt_letter.pdf](https://github.com/user-attachments/files/19575348/PDFKit_one_debt_letter.pdf)

 @sara-amanda @EvanAtCoforma @lakper

**Comment by lakper** [comment]
> Hi [@SarahKay8](https://github.com/SarahKay8)
> 
> [@digitaldrk](https://github.com/digitaldrk) and [@Dr-Pongo](https://github.com/Dr-Pongo) worked on a new version of this PDF using PDFKit. Can you take a look at this version? Thank you
> 
> [PDFKit_one_debt_letter.pdf](https://github.com/user-attachments/files/19575348/PDFKit_one_debt_letter.pdf)
> 
> [@sara-amanda](https://github.com/sara-amanda) [@EvanAtCoforma](https://github.com/EvanAtCoforma) [@lakper](https://github.com/lakper)

Created a new intake ticket as 104267 is closed.

**Comment by SarahKay8** [comment]
I’m on it @mgleo

**Comment by mgleo** [comment]
Thank you @SarahKay8! 
There are some details that are being added such as lines in between rows on tables and Rights and Responsibilities text after the tables. But it is, for the most part, almost complete. 
Our Midpoint Review is tomorrow so I can let them know if you find any major issues.

**Comment by SarahKay8** [comment]
@mgleo @digitaldrk 

![Image](https://github.com/user-attachments/assets/f3a0013b-fd55-457d-ae17-817dfb481ee2)

This looks good! So does the metadata. I would make these two separate tables and break the "To pay your copay bills" paragraphs out of the table. Doesn't seem like this info is apart of the table seems like this is standalone information. Other than that you're all set

**Comment by lakper** [comment]
@mgleo when you have a moment, we'd love to [hear your feedback](https://dj540s05.optimalworkshop.com/questions/accessibility-digital-experience-feedback-survey) on how we did!
